### PR TITLE
Added more info to the "nothing=/dev/null" option

### PR DESCRIPTION
**Pull request by bfallonf**
@mwringe As discussed via email. I've added the second para below (and edited the first).

Can I get an ack this is what you meant? Thanks!

### CODE REVIEW

**Comment by bfallonf** [comment]
@mwringe bump

**Comment by mwringe** [comment]
Looks good to me.

**Comment by bfallonf** [comment]
@mwringe Thanks!

@tpoitras @tnguyen-rh @adellape @ahardin-rh Any thoughts?

File: install_config/cluster_metrics.adoc [hunk 1]
```diff
@@ -165,10 +165,7 @@ values as link:../dev_guide/secrets.html[secrets] to the *metrics deployer*.
 
 [WARNING]
 ====
-By default, the Hawkular Metrics service uses self signed certificates which can
-cause problems when the console accesses it in a web browser. Alternately, you
-can specify your own *_hawkular-metrics.pem_* secret to use a certificate which
-is trusted by the browser.
+By default, the Hawkular Metrics service uses self-signed certificates, which can cause problems when the console accesses it in a web browser. Alternately, you can specify your own *_hawkular-metrics.pem_* secret to use a certificate that is trusted by the browser.
```
Comment (tnguyen-rh) [inline install_config/cluster_metrics.adoc]: s/Alternately/As an alternative/

File: install_config/cluster_metrics.adoc [hunk 2]
```diff
@@ -179,8 +176,7 @@ $ oc secrets new metrics-deployer hawkular-metrics.pem=/home/openshift/metrics/h
 hawkular-metrics-ca.cert=/home/openshift/metrics/hm-ca.cert
 ----
 
-You can also use the default auto-generated certificates with the following
-command:
+The Metrics Deployer can accept multiple certificates using secrets. If a certificate is not passed as a secret, the deployer will generate a self-signed certificate to be used instead. For the deployer to generate certificates for you, a secret is still required before it can be deployed. In this case, create a 'dummy' secret that does not specify a certificate value:
```
Comment (tnguyen-rh) [inline install_config/cluster_metrics.adoc]: Not sure about the single-quotes around "dummy".  I think either double-quotes or unquoted is better.  I often see it unquoted in technical docs.  It's not mentioned in [this writing style guide](https://obriend.fedorapeople.org/WritingStyleGuide/), which leads me to believe it's sufficiently mainstream.

**Comment by tnguyen-rh** [comment]
@bfallonf Two nits, otherwise LGTM.

File: install_config/cluster_metrics.adoc (hunk 2 above)
Comment (ahardin-rh) [inline install_config/cluster_metrics.adoc]: +1 on the "dummy" note. Other than that, looks good to me!

**Comment by bfallonf** [comment]
@tnguyen-rh @ahardin-rh Thanks a bunch. Suggestions taken in. You're both great. Merging.

**Comment by adellape** [comment]
@bfallonf I removed the `dedicated-3.1` label from this PR because the Dedicated build doesn't have any of the admin-focused guides (like the Install & Guide this PR updates). Also removed the `enterprise-3.0` label because this metrics content does not appear until the 3.1 build.

Also noticed the updated paragraphs are not wrapped, but, not a big deal.

**Pull request merged**